bitrise.yml

```diff
@@ -231,6 +231,39 @@ workflows:
     - opts:
         is_expand: false
       INTEGRATOR_VARIANT: debug
+  upgrade_dependencies:
+    steps:
+    - activate-ssh-key@4: {}
+    - git-clone@6: {}
+    - cache-pull@2: {}
+    - install-missing-android-tools@3:
+        inputs:
+        - gradlew_path: $PROJECT_LOCATION/gradlew
+    - gradle-runner@2:
```

Comment:
I think it would be nice if you add some meaningful titles to this and the next step to make it easier to read. You can find example in this file